Products/ZenUI3/browser/resources/js/zenoss/form/graphPanel.js

```diff
@@ -709,6 +713,7 @@
             if(!ms){
                 return;
             }
+            isAdjusted = isAdjusted || true;
```

Comment:
Uhm. Unintentionally yes. Code works because argument is not being passed at all in some usages, so `isAdjusted` will be `undefined` and default to _true_.

Anyway, this is a bug, let me fix it.